framework/doc/content/source/userobject/ActivateElementUserObject.md

```diff
@@ -0,0 +1,15 @@
+# ActivateElementUserObject
+
+This user object mimics the process of activating (adding) an element by moving the element from an "inactive" subdomain to the "active" subdomain. There are two metrics for activating an element, i.e., by +function path+ and by +variable value+.
+
+- Activation by +function path+ uses an user provided path function $f(x(t),y(t),z(t))$ and activates an element if this element is close to the location of the path, i.e., $f(x(t_0), ,y(t_0),z(t_0))$ at time $t_0$. The path function is set by `function_x`, `function_y`, and `function_z` in the input block.
```

Comment:
there is no _f_, this should be the "point `$(x(t), y(t), z(t))$` with components defined by the functions specified by the parameters `function_x`, `function_y`, and `function_z` in the input."